_extras/guide.md

```diff
@@ -23,109 +27,127 @@ Because this lesson follows a single dataset throughout, its easy for learners (
+
+* End each episode with a summary of what you just did and going over the best-practices. Take this as an opportunity to connect the skills used for this specific science case to the broader skill set that can be applied to any science case.
+
+* Share links to the episode with learners. There is a lot of typing and if a student gets stuck debugging a typo, having the episode can allow them to catch up. Encourage them to type along and only use the episode as a back up.
 
-* Errors will happen throughout the workshop. In an online setting it can be especially useful for learners to share their screen with the class why you troubleshoot together. This gives other learners the opportunity to see how you troubleshoot and to see common errors. However, be aware of the time and don't derail the entire lesson if someone is having an edge case issue.
+* It is recommended that at a minimum each day be a new notebook to avoid a single notebook that is so large that it become slow to interact with. While it would be great to demonstrate the best practice of breaking a complex analysis into multiple notebooks by starting each new episode in a new notebook, this will add overhead to your teaching time both to launch a new notebook and get it set up with the previously defined variables, so use your best judgement based on the level of your audience. If you decide to start each episode in a new notebook you can use the "Starting from this Episode" call out at the beginning of each episode to import and define everything you need from previous episodes. If you are using files from the `student_download` directory be sure to add the correct path `student_download/data/<filename>`.
 
-### Lesson 1: Queries
+* It is recommended that you do not repeat variable names and that you do not modify queries in place. If a student misses a step then they do not have an easy way to determine that they do not have the same code that you have.
 
-* At the beginning of this lesson, possible points of confusion are (1) there are two functions with similar named, `load_tables` and `load_table`, and neither of them actually downloads a table; rather, they download metadata about the tables.
+* Although learners should have seen functions in the SWC python episode, its always good to reiterate the process of prototyping what you want to do. Then if you find yourself wanting to do it over and over, make it into a function (a good rule of thumb is if you are copy and pasting, then think about whether you should be using a function).
 
-* Note that even though we don't use the parallax column for our analysis, we include it in the results for use in the data exploration section
+* Highlight throughout the episode what commands are ADQL specific and what are general SQL commands.
```

Comment:
Note: we'll probably want a guide for this if it isn't already clear in the notebook.

Reply:
I think its what is introduced in Lesson 2: polygon, contains, points, but we should double check

Reply:
It does list those three.

Reply:
I think the language is ambiguous enough in episode 1 that its worth listing explicitly, updating now